puppet/environments/seed_stack/manifests/site.pp

```diff
@@ -46,22 +46,28 @@
 # Keep track of node IP addresses across the cluster
 # FIXME: A better, more automatic way to do this
 class seed_stack_cluster {
-  $controller_ip = '192.168.0.2'
-  $worker_ip = '192.168.0.3'
-
-  host { 'controller.seed-stack.local':
-    ip           => $controller_ip,
-    host_aliases => ['controller'],
-  }
-  host { 'worker.seed-stack.local':
-    ip           => $worker_ip,
-    host_aliases => ['worker'],
-  }
+  $controller_ip = '192.168.55.11'
+  $worker_ip = '192.168.55.21'
+
+  # The hostmanager vagrant plugin manages the hosts entries for us.
+
+  # host { 'controller.seed-stack.local':
+  #   ip           => $controller_ip,
+  #   host_aliases => ['controller'],
+  # }
+  # host { 'worker.seed-stack.local':
+  #   ip           => $worker_ip,
+  #   host_aliases => ['worker'],
+  # }
 }
 
 node 'controller.seed-stack.local' {
   include seed_stack_cluster
 
+  file { ['/data/', '/data/brick1/', '/data/brick2']:
```

Comment:
It was needed, and it was in the standalone node. The controller/worker setup was broken.